src/Bridge/Elasticsearch/DataProvider/ItemDataProvider.php

```diff
@@ -56,6 +56,10 @@ public function __construct(Client $client, DocumentMetadataFactoryInterface $do
      */
     public function supports(string $resourceClass, ?string $operationName = null, array $context = []): bool
     {
+        if (\in_array($operationName, ['put', 'delete'], true)) {
```

Comment:
I don't think we can related on the operation name here as the route method can be different from the operation name no?

Reply:
I see your point. I can just rename the operation to "put2" via annotations for example and the fix won't work already. 

Option #2
Use item operation "method" attribute in the resource metadata. It's a bit better, but we can't rely on it either, since it's possible to omit it. You can just set "route_name" and there won't be "method" property.

Option #3
Another option would be to directly check current request method.

Reply:
Meh yeah you're right about option 2... @teohhanhui what would you do?